projects/exui-common-lib/src/lib/components/generic-filter/generic-filter.component.ts

```diff
@@ -297,8 +298,8 @@ export class GenericFilterComponent implements OnInit, OnDestroy {
       if (field.type === 'checkbox' || field.type === 'checkbox-large') {
         const formArray = this.buildCheckBoxFormArray(field, settings);
         this.form.addControl(field.name, formArray);
-      } else if (field.type === 'find-location') {
-        const formArray = this.buildFindLocationFormArray(field, settings);
+      } else if (field.type === 'find-location' || field.type === 'find-service') {
+        const formArray = this.buildFindLocationAndServiceFormArray(field, settings);
```

Comment:
can you please rename buildFindLocationAndServiceFormArray to something more generic if it fits multiple cases? (i.e. buildFormArray)

Reply:
Updated as suggested.